src/component/header.js

```diff
@@ -96,14 +97,15 @@ export const Title = ({ title = '', style, children }) => (
         style,
       ]}
     >
-      {title.toUpperCase()}
+      {keepCase ? title : title.toUpperCase()}
     </Text>
   </View>
 );
 
 Title.propTypes = {
   title: PropTypes.string,
   style: View.propTypes.style,
+  keepCase: PropTypes.bool,
```

Comment:
Changing Title shouldn't be necessary if we use a Text or Copy component.

Reply:
fixed